src/OrleansCodeGenerator/RoslynCodeGenerator.cs

```diff
@@ -89,8 +89,14 @@ public void GenerateAndLoadForAssemblies(params Assembly[] inputs)
             }
 
             var timer = Stopwatch.StartNew();
+            var emitDebugSymbols = false;
             foreach (var input in inputs)
             {
+                if (!emitDebugSymbols)
+                {
+                    emitDebugSymbols = RuntimeVersion.IsAssemblyDebugBuild(input);
```

Comment:
I've now changed it back, but it's redundant because if emitDebugSymbols was true, then this block would not execute. |= is clearer